build/Codecoverage.proj

```diff
@@ -31,19 +31,15 @@
       <_CodecovArgs Include="-t;$(CodeCovToken)" Condition="'$(CodeCovToken)' != ''" />
 
       <_CodecovFlags Include="$(Configuration)" Condition="'$(Configuration)' != ''" />
-      <_CodecovProductionFlags Include="@(_CodecovFlags)" />
-      <_CodecovProductionFlags Include="production" />
-      <_CodecovTestFlags Include="@(_CodecovFlags)" />
-      <_CodecovTestFlags Include="test" />
+      <_CodecovProductionAndTestFlags Include="@(_CodecovFlags)" />
+      <_CodecovProductionAndTestFlags Include="production" />
+      <_CodecovProductionAndTestFlags Include="test" />
```

Comment:
💡 You should be able to just use `_CodecovFlags`

```xml
<_CodecovFlags Include="production" />
<_CodecovFlags Include="test" />
```

Then replace the use of `@(_CodecovProductionAndTestFlags` below with `@(_CodecovFlags`.